dspfparser/src/main/kotlin/com/smeup/dspfparser/linesclassifier/DSPFRecord.kt

```diff
@@ -9,4 +9,5 @@ import kotlinx.serialization.Serializable
 sealed interface DSPFRecord {
     val name: String
     val fields: List<DSPFField>
+    val constants: List<DSPFField>
```

Comment:
I noticed that both constants and fields are instances of `DSPFField`. However, `DSPFField` includes properties like `x`, `y`, and `hasError`, which seem unrelated to constants. Is this a simplification to avoid creating a separate model class, or is this the correct model to use for constants as well?